### DISCUSSION TITLE
Connect hkknx to Jing ip router

### DISCUSSION THREAD

**Discussion by 3hajk** [brutella/hkknx-public#228] [Q&A] [answered]
Hi. I can't  connect hkknx app to my ip router from internet. I added port 3671 to nat rules in my router.  I have knx ip router by Jung. I imported my ETS proj. But I have many errors in console:
2022/11/09 23:28:45 group address GA-42 not found
2022/11/09 23:28:45 group address GA-4 not found
etc...

Can you help me?

**Comment by brutella** [comment, marked as answer]
Please try to explain your problem in more detail.

- Do you want to access your KNX IP Gateway from the Internet?
- Do you want to access the web ui of hkknx from the Internet?
- Do you want to access your HomeKit accessories from the Internet?

**Reply by 3hajk** [reply]
Hi. I want to access my KNX IP Gateway from Internet now. I have a working knx system at my house and server visualization on GIRA X1. But I want to connect to my system from apple home kit

**Reply by brutella** [reply]
Have you set the external IP address of your internet router in Setting → KNX Gateway?

**Reply by 3hajk** [reply]
Yes

**Reply by 3hajk** [reply]
and add to firewall rules port 3671

**Reply by brutella** [reply]
Have you set the protocol to TCP or UDP?

**Reply by 3hajk** [reply]
TCP only

**Reply by 3hajk** [reply]
I changing nat rule to udp and check now

**Reply by 3hajk** [reply]
app.go:325: dial gateway: no route to 109.xxx.xxx.xxx:3671

**Reply by brutella** [reply]
Are you able to connect to your KNX gateway from the ETS?

**Reply by 3hajk** [reply]
I dont have ETS.

**Reply by 3hajk** [reply]
My system is designed by another company

**Reply by brutella** [reply]
Please provide more details about your NAT settings.

**Reply by 3hajk** [reply]
Ok this will take some time. I don't have electricity now.

**Reply by 3hajk** [reply]
2    ;;; Gira X1
      chain=dstnat action=dst-nat to-addresses=192.168.XXX.XXX protocol=tcp 
      in-interface=ether1 dst-port=4432 log=no log-prefix="" 

 3    ;;; KNX IP router
      chain=dstnat action=dst-nat to-addresses=192.168.XXX.XXX protocol=udp 
      in-interface=ether1 dst-port=3671 log=no log-prefix=""

**Reply by 3hajk** [reply]
firewall  
;;; Gira X1
      chain=input action=accept protocol=tcp in-interface=ether1 dst-port=4432 
      log=no log-prefix="" 

 ;;; KNX IP router
      chain=input action=accept protocol=udp in-interface=ether1 dst-port=3671 
      log=no log-prefix=""
